netlify.toml

```diff
@@ -11,7 +11,7 @@
   publish = "website/build/testing-library-docs"
 
   # Default build command.
-  command = "cd website; npm install; npm run build"
+  command = "cd website; npm install; npm run format-docs; npm run build"
```

Comment:
Question! Won't this create a diferent version of deployed docs than the one on git/GitHub?

Reply:
Yeah, but it's pretty harmless I think. The alternative is to put the format command as a CI check, but it's hard to actually fix up prettier differences on GitHub.

Reply:
I figure we can periodically run the `format-docs` command to clean up.

Reply:
Yep, agree.. at the end of the day, people should just fork and edit docs locally, so git hooks autofix everything. But yeah, I can see how a "safety check" right before deploying could help. The only risk is that formatting ends up breaking the docs, but this feels kinda unlikely.

Reply:
Yeah, I would think the formatting would break the build rather than breaking the docs silently after the build.